<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="button_1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Test Button</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Herculanum</family>
     <pointsize>18</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>IF YOU SEE THIS IT WORKS</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>160</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="test_layout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
